PowerShell/JumpCloud Module/Tests/Public/Commands/Get-JCCommand.tests.ps1

```diff
@@ -39,6 +39,47 @@ Describe -Tag:('JCCommand') 'Get-JCCommand 1.0' {
         $Triggers = Get-JCCommand | Where-Object trigger -ne ''
         $Triggers._id.Count | Should -BeGreaterThan 1
     }
-
-
 }
+
+Describe -Tag('JCCommand') 'Get-JCCommand Search' {
```

Comment:
May need to add the -returnProperties check if we decide to include that functionality